backend/auth/test/helpers/db.helper.ts

```diff
@@ -50,6 +51,16 @@ export async function getUserByAttribute(
   );
 }
 
+export async function getLookupsByUserId(dataSource: DataSource, fkUserId: number) {
+  return await getRepository<ImageUserLookupEntity>(dataSource, ImageUserLookupEntity).findBy(
+    { fkUserId },
+  );
+}
+
+export async function createLookup(dataSource: DataSource, lookup: ImageUserLookupEntity) {
```

Comment:
should we also name it insertLookup? since creating doesn't imply having it in the databse.
Someone said thats what bigger companies do but idc

Reply:
Might be...in this case I used 'create' to be uniform with the createUser function